.ci/linux-steps.yaml

```diff
@@ -7,6 +7,11 @@ steps:
 # Install Build Dependencies.
 # Install mlpack from master to get new functionalities instantly.
 - script: |
+    # Install cereal.
+    wget https://github.com/USCiLab/cereal/archive/v1.3.0.tar.gz
+    tar -xvzpf v1.3.0.tar.gz # Unpack into cereal-1.3.0/.
+    cd cereal-1.3.0/
```

Comment:
Maybe this `cd` isn't necessary?

Reply:
Nice catch, this should hopefully also fix the linux build.